### PR TITLE
Fix/composer json

### PR DESCRIPTION
**Pull request by szepeviktor**
See the commits!

@Mte90 ~~Please consider directing user to PHAR distribution of WP-CLI. It creates dependency problems.~~
See https://github.com/lucatume/wp-browser/issues/431

### CODE REVIEW

File: plugin-name/composer.json [hunk 1]
```diff
@@ -33,8 +34,7 @@
       "yahnis-elsts/plugin-update-checker": "^4.9"
    },
    "require-dev": {
-      "szepeviktor/phpstan-wordpress": "^0.6.0",
-      "szepeviktor/phpstan-phpdoc": "^0.2.1",
```
Comment (Mte90) [inline plugin-name/composer.json:37]: Why did you removed that package? It is not now integrated in the other one?

File: plugin-name/composer.json [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
    "description": "Wordpress Plugin Boilerplate with steroids!",
+   "type": "wordpress-plugin",
    "license": "GPL-2.0-or-later",
    "authors": [
       {
          "name": "author_name",
-         "email": "author@email.it",
-         "homepage": "http://author.url"
+         "email": "author@example.com",
+         "homepage": "https://author-url.test"
```
Comment (Mte90) [inline plugin-name/composer.json:11]: This change will require to update the generator https://github.com/WPBP/generator/blob/ed2fed4d13acaf2c94eb740cb982fa944c5cdd38/generator/composer.php#L224
Also for the email, can you do a pr also for that?

File: plugin-name/composer.json (hunk 1 above)
Comment (szepeviktor) [reply plugin-name/composer.json:37]: phpstan-phpdoc is an excerpt of PHPStan's documentation-related rules: no new code there.

File: plugin-name/composer.json (hunk 2 above)
Comment (szepeviktor) [reply plugin-name/composer.json:11]: Done.
https://github.com/WPBP/generator/pull/46

**Pull request merged**